File: q9.cpp
#include <iostream>
using namespace std;
int main(){
    float fruta;
    float pagamento;
    float custo;
    cout<<"Digite a quantidade de frutas compradas: ";
    cin>>fruta;
    if (fruta>=12){
        custo=fruta*1.9;
        cout<<custo;
    }else{
        pagamento=fruta*2.5;
        cout<<pagamento;
    }
    return 0;
    }

File: q9.py
fruta=float(input("digite o numero de frutas compradas: "))
custo=(fruta*1,90)
pagamento=(fruta*2,50)
if fruta>=12:
    custo=(fruta*1)
    print(custo)
else:
    pagamento=(fruta*1.3)
    print(pagamento)
